### PR TITLE
created app Bet first part

### PR DESCRIPTION
**Pull request by Alekearl**

### CODE REVIEW

File: src/main/java/core/basesyntax/dao/UserDaoImpl.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+public class UserDaoImpl implements UserDao {
+    @Override
+    public void add(User user) {
+        Storage.users.add(user);
+    }
+
+    @Override
+    public List<User> getAll() {
+        return null;
```
Comment (oleksandrdziubiuk) [inline src/main/java/core/basesyntax/dao/UserDaoImpl.java]: return Storage.users;

**Review by oleksandrdziubiuk — APPROVED** [review]

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.model.Bet;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    BetDao betDao = new BetDaoImpl();
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: what is missing?

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+            if(command.equalsIgnoreCase("q")) {
+                return;
+            }
+            Bet bet = null;
+            try {
+                String[] betData = command.split(" ");
+                int value = Integer.parseInt(betData[0]);
+                double risk = Double.parseDouble(betData[1]);
+                bet = new Bet(value, risk);
+            } catch (NumberFormatException e) {
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: what about ArrayIndexOutOfBoundException? let’s catch it here as well

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+                System.out.println("Помилка. Будь ласка, введіть корректні дані.");
+            }
+
+            betDao.add(bet);
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: let’s save  the bet if it is not null

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+            System.out.println(bet == null ? null : bet.toString());
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]:
```suggestion
            System.out.println(bet);
```

File: src/main/java/core/basesyntax/dao/UserDaoImpl.java (hunk 1 above)
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/dao/UserDaoImpl.java]: let's return the correct value

File: src/main/java/core/basesyntax/model/User.java [hunk 6]
```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.model;
+
+public class User {
+    private String userName;
+    private int userAge;
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/model/User.java]:
```suggestion
    private String name;
    private int age;
```

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java (hunk 2 above)
Comment (Alekearl) [reply src/main/java/core/basesyntax/controller/ConsoleHandler.java]: added.

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java (hunk 3 above)
Comment (Alekearl) [reply src/main/java/core/basesyntax/controller/ConsoleHandler.java]: solved

File: src/main/java/core/basesyntax/dao/UserDaoImpl.java (hunk 1 above)
Comment (Alekearl) [reply src/main/java/core/basesyntax/dao/UserDaoImpl.java]: changed. ^__^

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java (hunk 4 above)
Comment (Alekearl) [reply src/main/java/core/basesyntax/controller/ConsoleHandler.java]: changed

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**